engines/cine/metaengine.cpp

```diff
@@ -245,6 +251,170 @@ SaveStateDescriptor CineMetaEngine::querySaveMetaInfos(const char *target, int s
 	return SaveStateDescriptor();
 }
 
+Common::KeymapArray CineMetaEngine::initKeymaps(const char *target) const {
+	using namespace Common;
+	using namespace Cine;
+
+	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "cine-default", _("Default keymappings"));
+	Keymap *gameKeyMap = new Keymap(Keymap::kKeymapTypeGame, "game-shortcuts", _("Game Keymappings"));
+	Keymap *mouseKeyMap = new Keymap(Keymap::kKeymapTypeGame, "mouse-shortcuts", _("Key to Mouse Keymappings"));
+
+	Action *act;
+
+	act = new Action(kStandardActionLeftClick, _("Left click"));
+	act->setLeftClickEvent();
+	act->addDefaultInputMapping("MOUSE_LEFT");
+	act->addDefaultInputMapping("JOY_A");
+	engineKeyMap->addAction(act);
+
+	act = new Action(kStandardActionRightClick, _("Right click"));
+	act->setRightClickEvent();
+	act->addDefaultInputMapping("MOUSE_RIGHT");
+	act->addDefaultInputMapping("JOY_B");
+	engineKeyMap->addAction(act);
+
+	act = new Action("MOUSELEFT", _("Select option / Click in game"));
+	act->setCustomEngineActionEvent(kActionMouseLeft);
+	act->addDefaultInputMapping("RETURN");
+	act->addDefaultInputMapping("KP_ENTER");
+	act->addDefaultInputMapping("KP5");
+	mouseKeyMap->addAction(act);
+
+	act = new Action("MOUSERIGHT", _("Open action menu / Close menu"));
+	act->setCustomEngineActionEvent(kActionMouseRight);
+	act->addDefaultInputMapping("ESCAPE");
+	mouseKeyMap->addAction(act);
```

Comment:
Check CineEngine::showSplashScreen() - I think the mouseKeyMap might block pressing escape to skip the splash screen